xwalk.gyp

```diff
@@ -632,8 +634,11 @@
       ],
       'copies': [
```

Comment:
Are you sure this is still needed (ie. changing the output directory above in `xwalk_strings` is not enough)?